pkgs/build-support/node/build-npm-package/default.nix

```diff
@@ -22,14 +22,18 @@
 , npmBuildScript ? "build"
   # Flags to pass to all npm commands.
 , npmFlags ? [ ]
-  # Flags to pass to `npm ci` and `npm prune`.
+  # Flags to pass to `npm ci`.
 , npmInstallFlags ? [ ]
   # Flags to pass to `npm rebuild`.
 , npmRebuildFlags ? [ ]
   # Flags to pass to `npm run ${npmBuildScript}`.
 , npmBuildFlags ? [ ]
   # Flags to pass to `npm pack`.
 , npmPackFlags ? [ ]
+  # Flags to pass to `npm prune`.
+, npmPruneFlags ? npmInstallFlags
+  # Value for npm `--workspace` flag and directory in which the files to be installed are found.
+, npmWorkspace ? null
```

Comment:
Would you mind adding these arguments to [the documentation](https://github.com/NixOS/nixpkgs/blob/3adc3587e8e8ecbe90153e7ad4e7b3b60b592289/doc/languages-frameworks/javascript.section.md#arguments-javascript-buildnpmpackage-arguments)?

Besides that. LGTM